Sources/UtilityBeltData/CoreData/CoreDataOperator+Mocked.swift

```diff
@@ -9,8 +9,11 @@
         /// Creates a mocked `CoreDataOperator` for use with testing.
         /// - Parameter name: The name of the Data Model to persist.
         /// - Parameter storeType: The method of storage for this `CoreDataOperator`. Defaults to In Memory.
-        static func mocked(name: String, storeType: NSPersistentContainer.StoreType = .memory) -> CoreDataOperator {
-            return CoreDataOperator(persistentContainer: .mocked(name: name, storeType: storeType))
+        /// - Parameter managedObjectModel: The managed Data Model to persist if mocking programmatically.
```

Comment:
> if mocking programmatically

Could you explain what "mocking programmatically" means? Or maybe even what the opposite of "programmatic" would be?